README.md

```diff
@@ -15,3 +15,7 @@ API for registration, live-site
 ## How to use and contribute
 
 See documentation here: <https://docs.mchacks.ca>
+
+## Deploy
+
+[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy)
```

Comment:
is the button supposed to link to our Heroku app?

Reply:
No. This is a deploy to Heroku button that creates a new instance of the API.